syntax = "proto2";

enum LEDDescT
{
    LED_POST          = 0;
    LED_STATUS_GREEN  = 1;
    LED_STATUS_YELLOW = 2;
}

enum LEDStateT
{
    LED_OFF = 0;
    LED_ON = 1;
}
message LEDRequest
{
    required LEDDescT  led = 1;
    required LEDStateT state = 2;
}

message LEDResponse
{
    required bool success  = 1;
	optional string errorMessage = 2 [default = ""];
}